ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Prerequisites
+
+* [ ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+repo%3Aatom%2Felectron) to see if it is already reported?
+
+### Description
+
+[Description of the bug or feature, for example, what you were doing when the issue arose and what you expected to happen]
+
+Electron version: 
```

Comment:
Probably move electron version and OS above `[Description of bug]`, the description can be quite long.